docs/sql-ref-functions-builtin-aggregate.md

```diff
@@ -19,4 +19,616 @@ license: |
   limitations under the License.
 ---
 
-Aggregate functions
\ No newline at end of file
+Spark SQL provides build-in aggregate functions defined in the dataset API and SQL interface. Aggregate functions
+operate on a group of rows and return a single value.
+
+Spark SQL aggregate functions are grouped as <code>agg_funcs</code> in Spark SQL. Below is the list of functions.
+
+**Note:** All functions below have another signature which takes String as a expression.
+
+<table class="table">
+  <thead>
+    <tr><th style="width:25%">Function</th><th>Parameter Type(s)</th><th>Description</th></tr>
```

Comment:
nit: as the Pg doc does so, I like `Argument` better than `Parameter`.

Reply:
ok